code/datums/ai_law_sets.dm

```diff
@@ -25,7 +25,7 @@
 
 /******************** Nanotrasen/Malf ********************/
 /datum/ai_laws/nanotrasen
-	name = "NT Default"
+	name = "NT Recommended"
```

Comment:
Hmmm, its laws are: Safeguard, serve, protect, survive... 2 of them are the US police motto `to protect and serve` (if I recall correctly)... and three of them mention 'space station'

What about: `StationGuard`, `NT Defensive`, `Safeguard`, `NT Safe`... hmmm its difficult, indeed!